daemon/network.go

```diff
@@ -3,14 +3,14 @@ package daemon
 import (
 	"fmt"
 	"net"
-	"sort"
 	"strings"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/errors"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/network"
 	clustertypes "github.com/docker/docker/daemon/cluster/provider"
+	"github.com/docker/docker/pkg/plugins"
```

Comment:
We should stop importing `pkg/plugins` since that's pluginv1 only. Use of `pkg/plugingetter` with an object that implements the PluginGetter interface such as daemon.pluginStore is the right way to get plugins v2 and fallback to v1.